assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -49,19 +49,7 @@ function APIProvider({ children }) {
     /**
      * Fire REST API call to save story.
      *
-     * @param {Object} 	 story - A story object.
-     * @param {number}   story.storyId Story post id.
-     * @param {string}   story.title Story title.
-     * @param {string}   story.status Post status, draft or published.
-     * @param {Array}    story.pages Array of all pages.
-     * @param {number}   story.author User ID of story author.
-     * @param {string}   story.slug   The slug of the story.
-     * @param {string}   story.date   The publish date of the story.
-     * @param {string}   story.modified   The modified date of the story.
-     * @param {string}   story.content AMP HTML content.
-     * @param {string}   story.excerpt Short description.
-     * @param {number}   story.featuredMedia Featured image id.
-     * @param {string}   story.password Password
+     * @param {import('../../types').Story} story Story object.
```

Comment:
?

Reply:
Imports the type definition from the `types.js` file so the editor knows the shape of `story`.